Update GCP uninstall intructions (#96)
ansible/aap-docs: stories/topics/proc-aap-gcp-delete-upgrade-secrets.adoc

[id="proc-aap-gcp-delete-upgrade-secrets"]

> = Deleting secrets left over after upgrade and backup restore

Reviewer: ```suggestion
= Deleting secrets left over after upgrade and backup and restore
```
Not sure if this suggestion would work

[NOTE]
==== 
This action cannot be reversed.

Only perform this action if your deployment is deleted.
====

> There are a number of possible secret names you can find, depending on the procedures you performed before:

Reviewer: ```suggestion
There are several possible secret names you can find, depending on the procedures you have completed:
```
